src/views/Question.vue

```diff
@@ -197,7 +210,6 @@ watch:{
   color: white;
   border-radius: 10000px;
   margin-top: 60px;
```

Comment:
Instead of using `buttonState` for switching between `.button-enabled` and `.button-disabled` class names, you could try apply the "enabled" state for the `button` by default, and style the "disabled" state by reusing back the `disabled` property, for example:

```css
button {
  /* default enabled styles */
}
button[disabled] {
  /* disabled styles */
}
```

Reply:
If I style the "button", it will affect all buttons in the whole project, right? In that case, should I put a class on this button?

Reply:
@vancass if you use `<style scoped>` (which I think we should do for *all* files, to be safe), by right it won't affect other pages.

Reply:
@cheeaun alright I'll read more about `<style scoped>`